LayoutTests/compositing/animation/hidden-animated-layer-should-not-have-scrollbars.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE HTML>
```

Comment:
How does this test case work? It doesn't dump layers.

Reply:
As discussed, this is just correctness fix rather than fixing bug, which needs to be fixed separately.

Reply:
@smfr - update and all green EWS. :-) Thanks for your insight.